dev/how-to/deploy/orchestrated/ansible.md

```diff
@@ -765,109 +859,125 @@ Edit the `inventory.ini` file and add the following host variable after the IP o
 
-Log in to the Control Machine using the `tidb` user account, and log in to the IP of the target machine using SSH. If you do not need to enter the password and can successfully log in, then the SSH mutual trust is successfully configured.
+    {{< copyable "shell-regular" >}}
 
-```shell
-ssh 172.16.10.61
-```
+    ```shell
+    ssh 172.16.10.61
+    ```
 
-```
-[tidb@172.16.10.61 ~]$
-```
+    ```
+    [tidb@172.16.10.61 ~]$
+    ```
 
-After you login to the deployment target machine using the `tidb` user, run the following command. If you do not need to enter the password and can switch to the `root` user, then sudo without password of the `tidb` user is successfully configured.
+5. After you login to the deployment target machine using the `tidb` user, run the following command. If you do not need to enter the password and can switch to the `root` user, then sudo without password of the `tidb` user is successfully configured.
 
-{{< copyable "shell-regular" >}}
+    {{< copyable "shell-regular" >}}
 
-```shell
-sudo -su root
-```
+    ```shell
+    sudo -su root
+    ```
 
-```
-[root@172.16.10.61 tidb]#
-```
+    ```
+    [root@172.16.10.61 tidb]#
+    ```
 
 ### Error: You need to install jmespath prior to running json_query filter
 
-See [Install Ansible and its dependencies on the Control Machine](#step-4-install-ansible-and-its-dependencies-on-the-control-machine) and use `pip` to install Ansible and the related specific dependencies in the Control Machine. The `jmespath` dependent package is installed by default.
+1. See [Install Ansible and its dependencies on the Control Machine](#step-4-install-ansible-and-its-dependencies-on-the-control-machine) and use `pip` to install Ansible and the corresponding dependencies in the Control Machine. The `jmespath` dependent package is installed by default.
 
-Enter `import jmespath` in the Python interactive window of the Control Machine.
+2. Run the following command to check whether `jmespath` is successfully installed:
 
-- If no error displays, the dependency is successfully installed.
-- If the `ImportError: No module named jmespath` error displays, the Python `jmespath` module is not successfully installed.
+    {{< copyable "shell-regular" >}}
 
-```shell
-python
-```
+    ```bash
+    pip show jmespath
+    ```
 
-```
-Python 2.7.5 (default, Nov  6 2016, 00:28:07)
-[GCC 4.8.5 20150623 (Red Hat 4.8.5-11)] on linux2
-Type "help", "copyright", "credits" or "license" for more information.
->>> import jmespath
-```
+3. Enter `import jmespath` in the Python interactive window of the Control Machine.
+
+    - If no error displays, the dependency is successfully installed.
+    - If the `ImportError: No module named jmespath` error displays, the Python `jmespath` module is not successfully installed.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    python
```

Comment:
@liubo0127 Is running the `python` command here a necessary step? Why?

Reply:
Yes, this step confirms whether the dependent package can be used normally.

Reply:
Got it. Thanks!